README.md

```diff
@@ -161,23 +161,19 @@ import (
 	"log/slog"
 
 	fdk "github.com/CrowdStrike/foundry-fn-go"
-	"github.com/CrowdStrike/gofalcon/falcon"
-	"github.com/CrowdStrike/gofalcon/falcon/client"
+	"github.com/crowdstrike/gofalcon/falcon"
+	"github.com/crowdstrike/gofalcon/falcon/client"
 )
 
 func newHandler(_ context.Context, _ *slog.Logger, cfg config) fdk.Handler {
 	mux := fdk.NewMux()
 	mux.Post("/echo", fdk.HandlerFn(func(ctx context.Context, r fdk.Request) fdk.Response {
 		client, err := newFalconClient(ctx, r.AccessToken)
 		if err != nil {
-			if err == falcon.ErrFalconNoToken {
-				// not a processable request
-				return fdk.Response{ /* snip */ }
-			}
-			// some other error - see gofalcon documentation
+			return fdk.Response{ Code: 500, Body: fdk.JSON(err) }
```

Comment:
What's the motivation for returning a jsonified err? I think we're better off sticking with the `fdk.ErrResp(fdk.APIError{Code: 500, Message: err.Error()}`. This ties into fusino's UI as well as align with the idiomatic error handling of errors in a funciton.

Reply:
unfortunately, looking at teh sample apps for foundry, we don't have funcitons that utilize even a small percentage of our best practices. They need some TLC, so things like `fdk.ErrResp`, `fdk.HandleFnOf`, logging, etc are in plain view